tracer/src/Datadog.Trace.Tools.Runner/CheckAgentCommand.cs

```diff
@@ -0,0 +1,54 @@
+// <copyright file="CheckAgentCommand.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+#nullable enable
+
+using System;
+using System.Threading.Tasks;
+using Datadog.Trace.Agent;
+using Datadog.Trace.Configuration;
+using Datadog.Trace.Tools.Runner.Checks;
+using Spectre.Console;
+using Spectre.Console.Cli;
+
+namespace Datadog.Trace.Tools.Runner
+{
+    internal class CheckAgentCommand : AsyncCommand<CheckAgentSettings>
+    {
+        public override async Task<int> ExecuteAsync(CommandContext context, CheckAgentSettings settings)
+        {
+            ExporterSettings configuration;
+
+            if (settings.Url == null)
+            {
+                // Try to autodetect the agent settings
+                configuration = new ExporterSettings(new EnvironmentConfigurationSource());
+
+                AnsiConsole.WriteLine($"No agent url provided, using environment variables");
```

Comment:
```suggestion
                AnsiConsole.WriteLine($"No Agent URL provided, using environment variables.");
```

Reply:
Done in https://github.com/DataDog/dd-trace-dotnet/pull/2337/commits/ed76a930c03fe7331b3bcbcb2b619abadfdb39c2